SimCalorimetry/EcalEBTrigPrimAlgos/src/EcalEBTrigPrimTestAlgo.cc

```diff
@@ -225,14 +232,14 @@ void EcalEBTrigPrimTestAlgo::run(const edm::EventSetup & setup,
 	  int nSam=0;
 	  for (int iSample=firstSample;iSample<=lastSample;++iSample) {
 	    etInADC= tcpformat_out_[iSample];
-	    if (debug_) std::cout << " format_out " << tcpformat_out_[iSample] <<  " etInADC " << etInADC << std::endl;
+	    //if (debug_) std::cout << " format_out " << tcpformat_out_[iSample] <<  " etInADC " << etInADC << std::endl;
```

Comment:
delete commented-out code

Reply:
Please do, or justify why you prefer not doing it